ipaserver/install/adtrustinstance.py

```diff
@@ -130,6 +130,15 @@ def map_Guests_to_nobody():
     logger.debug("Map BUILTIN\\Guests to a group 'nobody'")
     ipautil.run(args, env=env, raiseonerr=False, capture_error=True)
 
+
+def get_idmap_range(realm):
+    range = api.Command.idrange_show('%s_id_range' % realm)['result']
+    range_start = int(range['ipabaseid'][0])
+    range_size = int(range['ipaidrangesize'][0])
+    range_fmt = '%s - %s' % range_start, range_start + range_size
```

Comment:
``TypeError: not enough arguments for format string``

Reply:
Fixed by using format()